getting-started/quick-start.md

```diff
@@ -0,0 +1,110 @@
+# Getting Started with Amber
+This getting started guide will help you get a full stack web application running in just a few minutes by leveraging amber’s code generators.
+
+After we have a working application, then we will walk through the code that is making it all happen and code some enhancements along the way.
+
+Let’s get started!
+
+## Install dependencies
+If you already have `crystal`, `amber`, and a database installed, you can skip this step.
+
+### Install `crystal`
+
+#### OS X with homebrew
+ ```
+$ brew install crystal-lang
+```
+#### Ubuntu or Debian
+ ```
+$ curl https://dist.crystal-lang.org/apt/setup.sh | sudo bash
+$ sudo apt-get install build-essential crystal
+```
+
+#### Full Install Instructions
+See full installation instructions [here](https://crystal-lang.org/docs/installation/)
+
+### Install `amber`
+
+#### OS X with homebrew
+```
+brew install amberframework/amber/amber
+```
+
+#### Ubuntu or Debian
+```
+$ sudo apt-get install build-essential libreadline-dev libsqlite3-dev libpq-dev libmysqlclient-dev libssl-dev libyaml-dev
+$ curl -L https://github.com/amberframework/amber/archive/stable.tar.gz | tar xz
+$ cd amber-stable/
+$ shards install
+$ make install
+```
+
+#### Full Install Instructions
+See full installation instructions [here](https://amberframework.org/guides/getting-started/Installation/README.md#installation)
+
+### Install a database
+Amber works with `postgresql` (default), `mysql`, or `sqlite`.
+
+If you don’t already have one of these installed, please follow the guides provided by each database maintainer:
+[Postgresql Installation Guides](https://wiki.postgresql.org/wiki/Detailed_installation_guides)
+On OS X postgres can also be installed with homebrew or the Postgres.app
+[MySQL Installation Guides](https://dev.mysql.com/doc/refman/8.0/en/installing.html)
+On OS X mysql can also be installed with homebrew
+[Tutorial on installing SQLite](https://www.tutorialspoint.com/sqlite/sqlite_installation.htm)
+On OS X sqlite3 can also be installed with homebrew
+
+## Generate a new Amber application
+With all dependencies successfully installed, we can generate a new application with `amber new`
+
+After the code for the new application is generated, we will `cd` into the new directory and execute a `shards install`.
+
+The shards install command may take a little while - it has to download all shard dependencies.
+```
+# The default setup will use a postgresql database, use `-d mysql` or `-d sqlite` for mysql and sqlite, respectively
+$ amber new pet-tracker
```

Comment:
I don't think we should put `$` before our lines as it causes annoyances when copying commands.

Reply:
I agree for a lot of this cases, for a couple in this PR, The `$` can be helpful to differentiate from comments and "output"

Also, what do you think of having the placeholder there for output, instead of the actual output?

My thought is that it may get out of date quickly, and it isn't that helpful for the quickstart guide - they should see it in their terminal.